tools/delete_candidate.php

```diff
@@ -189,6 +215,12 @@ function deleteCandidate($CandID, $PSCID, $confirm, $printToSQL, $DB, $output)
         //delete from parameter_candidate
         $DB->delete("parameter_candidate", array("CandID" => $CandID));
 
+        //delete from SNP_candidate_rel
+        $result = $DB->delete("SNP_candidate_rel", array("CandID" => $CandID));
+
+        //delete from CNV
```

Comment:
@driusan is it alright that these delete statements for added tables is contained in this PR? in the end, only these two `SNP_candidate_rel` and `CNV` needed to be updated